pkg/controller/deployment_rolling_test.go

```diff
@@ -116,7 +116,8 @@ var _ = Describe("deployment_rolling", func() {
 				for _, expectedNode := range data.expect.nodes {
 					actualNode, err := controller.targetCoreClient.CoreV1().Nodes().Get(expectedNode.Name, metav1.GetOptions{})
 					Expect(err).ToNot(HaveOccurred())
-					Expect(actualNode.Spec.Taints).Should(Equal(expectedNode.Spec.Taints))
+					Expect(actualNode.Spec.Taints).Should(HaveLen(len(expectedNode.Spec.Taints)))
```

Comment:
Length check is [built into](https://github.com/onsi/gomega/blob/master/matchers/consist_of.go#L42) `ConsistOf`. So additional length check is not required.